File: 프로그래머스/lv1/12912. 두 정수 사이의 합/두 정수 사이의 합.js
function solution(a, b) {
   
    let sum = 0;
    
    if (a > b) {
        [a, b] = [b, a]; 
        
    } else if (a === b) {
        return a
    }
    
    for (let i = a; i <= b; i++) {
        sum += i;
    }
    
    return sum;
}

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.py
def solution(a, b):
    sum = 0
    
    if a > b:  # a가 큰 경우
        for i in range(b, a + 1):
            sum += i
            
    else:  # b가 큰 경우
        for i in range(a, b + 1):
            sum += i
            
    return sum